doc/source/rllib/rllib-training.rst

```diff
@@ -574,10 +574,10 @@ The following is a list of the common algorithm hyper-parameters:
     #  - Any subclass of OffPolicyEstimator, e.g.
     #    ray.rllib.offline.estimators.is::ImportanceSampling or your own custom
     #    subclass.
-    "off_policy_estimation_methods": [
-        ImportanceSampling,
-        WeightedImportanceSampling,
-    ],
+    "off_policy_estimation_methods": {
```

Comment:
Should this be better:

```
"off_policy_estimation_methods": {
  "is": {
    "type": ImportanceSampling,
    [room for possible c'top kwargs here],
  },
  "wis": {
    "type": WeightedImportanceSampling,
    [room for possible c'top kwargs here],
  }
}
```
?

Then in the results, the user would find the data under the self-defined keys "is" and "wis" (the user can change these).

Reply:
wait, I don't get it, why do we double specify the name? "is" and ImportanceSampling?